Source/WTF/wtf/DebugHeap.h

```diff
@@ -55,9 +55,11 @@ class DebugHeap {
 #endif
 };
 
-#define DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Type) \
+#define DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Type) DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER_AND_EXPORT(Type, WTF_EXPORT_PRIVATE)
+
+#define DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER_AND_EXPORT(Type, Export) \
```

Comment:
You'll also need this macro available in non MHB builds.

Reply:
Right. Need to add support for the non MHB #define